scripts/run_mission1_robustness.py

```
@@ -0,0 +1,504 @@
+    bridge_proc = detector_proc = None
+    mission_log = run_dir / "mission.log"
+    mission_returncode = None
+    started_at = time.time()
+    try:
+        bridge_proc = start_process(bridge_cmd, run_dir / "bridge.log", env, ROOT)
+        time.sleep(bridge_delay)
+        detector_proc = start_process(detector_cmd, run_dir / "detector.log", env, ROOT)
+        time.sleep(detector_delay)
+
+        with open(mission_log, "w", encoding="utf-8") as log_handle:
+            mission_proc = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)